packages/ffe-account-selector-react/less/account-suggestion-multi.less

```
@@ -79,25 +60,25 @@
 }
 
 @media (prefers-color-scheme: dark) {
-    .ffe-account-suggestion {
+    .ffe-account-suggestion-multi {
         background: @ffe-black-darkmode;
 
+        &__nomatches {
+            background: @ffe-farge-svart;
+            color: @ffe-farge-hvit;
```

Så du endra fra `@ffe-farge-hvit` til `@ffe-white-darkmode` på linje 12, så kan kanskje endre her også? Med mindre det egentlig skal være omvendt da :D 
```suggestion
            color: @ffe-white-darkmode;
```

---

Samme med linje 67 med svartfargen.